npm-prepare-release/bin/prepare-release.sh

```diff
@@ -81,7 +81,7 @@ echo "✅ Checked out git branch ($NEW_BRANCH)"
 
 # git commit
 echo_title "git commit"
-git commit -a -m "Commiting new version of package"
+git commit -a -m "chore: bump package version to ${NEW_VERSION}"
```

Comment:
Can we make the `chore` string optional via flag? Not all packages use this and having the string added to commits for packages not using this feature may be confusing.